### PR TITLE
Enable email report send functionlity for longvity test

### PR DESCRIPTION
**Pull request by aleksbykov**
Adding to each yaml config file for longevity jobs config
parameters:

send_email: true
email_recipients: ['qa@scylladb.com']

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I gave variables/functions meaningful self-explanatory names
- [ ] I didn't leave commented-out/debugging code
- [ ] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: test-cases/longevity/longevity-100GB-48h-cloud-CloudLimitedChaosMonkey-tls.yaml [hunk 1]
```diff
@@ -27,3 +27,6 @@ authenticator: 'PasswordAuthenticator'
 authenticator_user: cassandra
 authenticator_password: cassandra
 authorizer: 'CassandraAuthorizer'
+
+send_email: true
+email_recipients: ['qa@scylladb.com']
```
Comment (amoskong) [inline test-cases/longevity/longevity-100GB-48h-cloud-CloudLimitedChaosMonkey-tls.yaml]: We should not push internal list address to public repo?

File: test-cases/longevity/longevity-100gb-4h.yaml [hunk 2]
```diff
@@ -17,3 +17,6 @@ nemesis_interval: 5
 
 user_prefix: 'longevity-100gb-4h'
 space_node_threshold: 64424
+
+send_email: true
+email_recipients: ['qa@scylladb.com']
```
Comment (roydahan) [inline test-cases/longevity/longevity-100gb-4h.yaml]: Why adding this to all yamls and not adding to test_defaults?

Comment (aleksbykov) [reply test-cases/longevity/longevity-100gb-4h.yaml]: @roydahan For that moment we have template for report only for gemini and longevity.
we will add templates to other test types, then this config will be moved to defaults

Comment (roydahan) [reply test-cases/longevity/longevity-100gb-4h.yaml]: we have also for performance.
So, I'm not sure which ones doesn't have the report.
It's ok to put it in the defaults and set "false" on those we don't have.

Comment (aleksbykov) [reply test-cases/longevity/longevity-100gb-4h.yaml]: @roydahan adding to defaults config
send_email: false
email_recipients: ['qa@scylladb.com']

File: defaults/test_default.yaml [hunk 3]
```diff
@@ -35,3 +35,6 @@ cloud_credentials_path: '~/.ssh/support'
 backtrace_decoding: false
 
 logs_transport: "rsyslog"
+
+send_email: false
```
Comment (bentsi) [inline defaults/test_default.yaml]: @aleksbykov what Roy meant to set `send_email: true` and set false to all that don't support sending emails. Please fix. Thanks

**Comment by bentsi** [comment]
closing this (implemented #1372)

**Pull request closed without merging**